shared/editor/nodes/Node.ts

```diff
@@ -19,6 +19,10 @@ export default abstract class Node extends Extension {
     return {};
   }
 
+  get attrs(): NodeSpec["attrs"] {
+    return {};
+  }
```

Comment:
This is done to please `tsc` when accessing `attrs` on _our_ nodes, for example `headingNode.attrs.collapsed`.